File: BeecrowdProblem/Average 1.dart
import 'dart:io';

void main(){
  average();
}

void average() {

  var a = double.parse(stdin.readLineSync()!);
  var b = double.parse(stdin.readLineSync()!);
  var ans = (a*3.5+b*7.5)/(3.5+7.5);
  print("MEDIA = ${ans.toStringAsFixed(5)}");
}

File: BeecrowdProblem/Average 1.kt

fun main(args: Array<String>) {
    val input = Scanner(System.`in`)

    val a = input.nextDouble()
    val b = input.nextDouble()

    val ans = (a * 3.5 + b * 7.5) / (3.5 + 7.5)

    println("MEDIA = %.5f".format(ans))
}